packages/commitlint-config-jira/src/index.ts

```diff
@@ -0,0 +1,16 @@
+import { UserConfig } from '@commitlint/types';
+
+const config: UserConfig = {
+  extends: ['@commitlint/config-conventional'],
+  plugins: ['@bigcommerce/commitlint-plugin-jira'],
+  rules: {
+    'scope-case': [2, 'always', 'lower-case'],
+    // https://github.com/conventional-changelog/commitlint/issues/2631
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-expect-error
+    'subject-case': [2, 'never', ['start-case', 'pascal-case']],
```

Comment:
I submitted a [PR in `commitlint`](https://github.com/conventional-changelog/commitlint/pull/2670) to resolve this issue. Waiting for that to get merged.